app/src/ui/changes/no-changes.tsx

```diff
@@ -582,7 +582,7 @@ export class NoChanges extends React.Component<
       <>
         You have{' '}
         {aheadBehind.ahead === 1 ? 'one local commit' : 'local commits'} waiting
-        to be pushed to {isGitHub ? 'GitHub' : 'the remote'}
+        to be pushed to {isGitHub ? 'GitHub' : 'the remote'}.
```

Comment:
@Rexogamer Would you be so kind to make this change (above) under the branch in https://github.com/desktop/desktop/pull/8007 ? as a PR is already open for rewording the uncommitted changes text. 

Happy to give you the credit for it all 😄

Reply:
I don’t know if I can edit stuff in that PR

Reply:
@tierninho

Reply:
This PR directs to the branch that PR merges, so the changes in this will be in that if merged

Reply:
hmm, not sure. I went ahead and made the change.